### PR TITLE
Some improvements

### PR DESCRIPTION
**Pull request by matthewma7**
See commit comment for details

### CODE REVIEW

File: server/rest/geom.py [hunk 1]
```diff
@@ -36,16 +39,26 @@ def __init__(self):
         self.route('DELETE', (':geomId',), self.deleteGeom)
         self.route('GET', ('export', ':itemId',), self.exportKPF)
 
+    # The girder default serialization takes twice the time than this custom serializer. Since geom could be relatively big, it worth to use this custom serializer.
```
Comment (mwoehlke-kitware) [inline server/rest/geom.py]: Pedantic: `s/twice the time than/twice the time as/` and `s/it worth/it is worthwhile/`. Also, line length.

File: server/rest/geom.py [hunk 2]
```diff
@@ -36,16 +39,26 @@ def __init__(self):
...
         .errorResponse()
         .errorResponse('Read access was denied on the item.', 403)
     )
     @access.user
+    @rawResponse
     def getGeomsOfItem(self, item, params):
+        setResponseHeader('Content-Type', 'application/json')
         cursor = Geom().findByItem(item)
-        return list(cursor)
+        jsonString = self.JSONEncoder().encode(list(cursor))
```
Comment (mwoehlke-kitware) [inline server/rest/geom.py:62]: BTW, is there a reason to not just return this directly?

File: web_external/IndexView/style.styl [hunk 3]
```diff
@@ -39,7 +39,7 @@ fieldset
   flex-flow row
 
   .left-sidebar
-    width 280px
+    width 290px
```
Comment (mwoehlke-kitware) [inline web_external/IndexView/style.styl:42]: I would still very much prefer using font units here...

File: web_external/Viewer/index.js [hunk 4]
```diff
@@ -140,7 +140,7 @@ class Viewer extends PureComponent {
                                         g0
                                     }
                                 })}
-                                annotationDeleted={() => this.props.dispatch({
+                                deleteAnnotation={() => this.props.dispatch({
```
Comment (mwoehlke-kitware) [inline web_external/Viewer/index.js:143]: This is not consistent with other names, especially `annotationDrawn`. If we're going to make changes, I would prefer making a general change, not an incomplete change that decreases consistency.

File: web_external/util/annotationActivityParser.js [hunk 5]
```diff
@@ -124,6 +124,7 @@ class AnnotationActivityContainer {
             this._trackActivityMap.set(newTrackId, map);
             this._trackActivityMap.delete(trackId);
         }
+        return this.copy();
```
Comment (mwoehlke-kitware) [inline web_external/util/annotationActivityParser.js:127]: Good catch 🙂.

File: server/rest/geom.py (hunk 1 above)
Comment (matthewma7) [reply server/rest/geom.py]: Thanks for the grammar correction

File: server/rest/geom.py (hunk 2 above)
Comment (matthewma7) [reply server/rest/geom.py:62]: I think this one is ok either way, I don't see anything wrong with assigning a value then return

File: web_external/IndexView/style.styl (hunk 3 above)
Comment (matthewma7) [reply web_external/IndexView/style.styl:42]: Again, with the latest browser, I don't see benefit using em over px.

File: web_external/Viewer/index.js (hunk 4 above)
Comment (matthewma7) [reply web_external/Viewer/index.js:143]: I am naming this based on what actually happened from the redux store (Or VAUI) point of view. The difference here is for annotationDrawn, it is drawn with GeoJS, it is something already happened, In the store, you could choose to ignore the drawn result but can't revert the fact that an annotation was drawn. 
But for annotation delete, from the store's point of view,  it was a request to delete. Naming it annotationDeleted sounds like it already happened. 
It just happens that GeoJS internally call that annotation, VAUI calls the boxes annotations as well. So, I think I would rename annotationDrawn to rectangleDrawn, at least for now, it is the only kind we support.

File: server/rest/geom.py (hunk 2 above)
Comment (mwoehlke-kitware) [reply server/rest/geom.py:62]: It's not *wrong*, per se, it just feels gratuitous... hence why the original comment is phrased as a question.

**Comment by matthewma7** [comment]
@mwoehlke-kitware So, is there something else you want to be addressed? If not, could you approve this?

**Review by mwoehlke-kitware — APPROVED** [review]

**Comment by matthewma7** [comment]
@mwoehlke-kitware Cool, thanks for the reviewing.

**Pull request merged**